specs/engine.spec/src/main/scripts/io/aklivity/zilla/specs/engine/streams/application/reconfigure.create.via.http.basic.auth/server.rpt

```diff
@@ -0,0 +1,67 @@
+#
```

Comment:
It's good that we verified the engine is initializing the filesystem correctly.

However, we also need http authorization test in http filesystem itself.